testing/deployer/Makefile

```diff
@@ -0,0 +1,84 @@
+SHELL := /bin/bash
+
+.PHONY: noop
+noop:
+
+##@ Build
+
+.PHONY: tidy
+tidy: ## Run go mod tidy.
+	go mod tidy
+
+##@ Checks
+
+.PHONY: format
+format: ## Format the go files.
+	@for f in $$(find . -name '*.go' -print); do \
+		gofmt -s -w $$f ; \
+	done
+
+.PHONY: lint
+lint: ## Run the full linting rules.
+	golangci-lint run -v
+
+.PHONY: vet
+vet: ## Run go vet.
+	go vet ./...
+
+##@ Testing
+
+.PHONY: test
+test: ## Run all go tests.
+	go test ./...
+
+##@ Code Generation
+
+.PHONY: update-defaults ## Update default images versions for all relevant containers to the latest.
+update-defaults: update-consul update-envoy update-dataplane
+
+.PHONY: update-consul
+update-consul: ## Update the default images for Consul CE and Enterprise containers to the latest.
```

Comment:
I would much rather just have bash scripts. There's 0 benefit from using make here, and then you have to deal with make-in-bash crap like `$$`.

Also, can we get `//go:generate` for this?

Reply:
Switched to a bash script called by `go generate`